src/OpenTelemetry.Extensions.Hosting/OpenTelemetryServicesExtensions.cs

```diff
@@ -15,121 +15,114 @@
-        /// <item>
-        /// This is safe to be called multiple times. Only a single <see
-        /// cref="TracerProvider"/> will be created for a given <see
-        /// cref="IServiceCollection"/>.
-        /// </item>
-        /// <item>
-        /// This method should be called by application host code. Library
-        /// authors should call <see
-        /// cref="TracerProviderBuilderServiceCollectionExtensions.ConfigureOpenTelemetryTracing(IServiceCollection)"/>
-        /// instead.
-        /// </item>
-        /// </list>
-        /// </remarks>
-        /// <param name="services"><see cref="IServiceCollection"/>.</param>
-        /// <returns>Supplied <see cref="IServiceCollection"/> for chaining
-        /// calls.</returns>
-        public static IServiceCollection AddOpenTelemetryTracing(this IServiceCollection services)
-            => AddOpenTelemetryTracing(services, b => { });
-
-        /// <summary>
-        /// Configure OpenTelemetry and register a <see cref="IHostedService"/>
-        /// to automatically start tracing services in the supplied <see
-        /// cref="IServiceCollection" />.
-        /// </summary>
-        /// <remarks><inheritdoc
-        /// cref="AddOpenTelemetryTracing(IServiceCollection)"
-        /// path="/remarks"/></remarks>
-        /// <param name="services"><see cref="IServiceCollection"/>.</param>
-        /// <param name="configure">Callback action to configure the <see
-        /// cref="TracerProviderBuilder"/>.</param>
-        /// <returns>Supplied <see cref="IServiceCollection"/> for chaining
-        /// calls.</returns>
-        public static IServiceCollection AddOpenTelemetryTracing(this IServiceCollection services, Action<TracerProviderBuilder> configure)
-        {
-            Guard.ThrowIfNull(services);
-
-            services.ConfigureOpenTelemetryTracing(configure);
-
-            services.TryAddEnumerable(ServiceDescriptor.Singleton<IHostedService, TelemetryHostedService>());
+    /// <remarks>
+    /// Notes:
+    /// <list type="bullet">
+    /// <item>
+    /// This is safe to be called multiple times. Only a single <see
+    /// cref="TracerProvider"/> will be created for a given <see
+    /// cref="IServiceCollection"/>.
+    /// </item>
+    /// <item>
+    /// This method should be called by application host code. Library
+    /// authors should call <see
+    /// cref="OpenTelemetryDependencyInjectionTracingServiceCollectionExtensions.ConfigureOpenTelemetryTracerProvider(IServiceCollection, Action{IServiceProvider, TracerProviderBuilder})"/>
+    /// instead.
+    /// </item>
+    /// </list>
+    /// </remarks>
+    /// <param name="services"><see cref="IServiceCollection"/>.</param>
+    /// <returns>Supplied <see cref="IServiceCollection"/> for chaining
+    /// calls.</returns>
+    [Obsolete("Use the AddOpenTelemetry().WithTracing(configure).StartWithHost() pattern instead. This method will be removed in a future version.")]
```

Comment:
If we're firm on our decision to drop these when we release stable, then maybe it makes sense to be more specific.

```suggestion
    [Obsolete("Use the AddOpenTelemetry().WithTracing(configure).StartWithHost() pattern instead. This method will be removed prior to a stable release of OpenTelemetry.Extensions.Hosting.")]
```

Reply:
I would prefer to keep as-is so we don't lock ourselves into anything, but I don't feel strongly about it if you really want this! 😄

Reply:
Ok that's fine. I guess the important thing to me is that we discuss as a SIG to drop these or not prior to stable release. You and I have discussed dropping them, but I want to make sure other maintainers agree.

The upside to keeping them is that it allows folks to still work with the old API and make the migration when it's convenient for them. Though I think the downside outweighs the upside in the long run - i.e. more API surface area, albeit marked obsolete, can lead to end user confusion.